src/FondOfSpryker/Zed/OrderApi/Business/Model/OrderApi.php

```diff
@@ -2,108 +2,196 @@
-                $errors[] = $error->getMessage();
-            }
+    /**
+     * @throws
+     *
+     * @return \Generated\Shared\Transfer\PropelQueryBuilderColumnSelectionTransfer
+     */
+    protected function buildColumnSelection(): PropelQueryBuilderColumnSelectionTransfer
+    {
+        $columnSelectionTransfer = new PropelQueryBuilderColumnSelectionTransfer();
+        $tableColumns = SpySalesOrderTableMap::getFieldNames(TableMap::TYPE_FIELDNAME);
+
+        foreach ($tableColumns as $columnAlias) {
+            $columnTransfer = (new PropelQueryBuilderColumnTransfer())
+                ->setName(SpySalesOrderTableMap::TABLE_NAME . '.' . $columnAlias)
+                ->setAlias($columnAlias);
+
+            $columnSelectionTransfer->addTableColumn($columnTransfer);
+        }
+
+        return $columnSelectionTransfer;
+    }
+
+    /**
+     * @param \Propel\Runtime\ActiveQuery\ModelCriteria $query
+     * @param \Generated\Shared\Transfer\ApiCollectionTransfer $apiCollectionTransfer
+     * @param \Generated\Shared\Transfer\ApiRequestTransfer $apiRequestTransfer
+     *
+     * @throws \Symfony\Component\HttpKernel\Exception\NotFoundHttpException
+     *
+     * @return array|\Generated\Shared\Transfer\ApiCollectionTransfer
+     */
+    protected function addPagination(
+        ModelCriteria $query,
+        ApiCollectionTransfer $apiCollectionTransfer,
+        ApiRequestTransfer $apiRequestTransfer
+    ): ApiCollectionTransfer {
+        $query->setOffset(0)
+            ->setLimit(-1);
+
+        $total = $query->count();
+        $page = $apiRequestTransfer->getFilter()->getLimit() ? ($apiRequestTransfer->getFilter()->getOffset() / $apiRequestTransfer->getFilter()->getLimit() + 1) : 1;
+        $pageTotal = ($total && $apiRequestTransfer->getFilter()->getLimit()) ? (int)ceil($total / $apiRequestTransfer->getFilter()->getLimit()) : 1;
+
+        if ($page > $pageTotal) {
+            throw new NotFoundHttpException('Out of bounds.', null, ApiConfig::HTTP_CODE_NOT_FOUND);
+        }
+
+        $apiPaginationTransfer = (new ApiPaginationTransfer())
+            ->setItemsPerPage($apiRequestTransfer->getFilter()->getLimit())
+            ->setPage($page)
+            ->setTotal($total)
+            ->setPageTotal($pageTotal);
+
+        $apiCollectionTransfer->setPagination($apiPaginationTransfer);
+
+        return $apiCollectionTransfer;
+    }
+
+    /**
+     * @param int $id
+     *
+     * @throws \Spryker\Zed\Api\Business\Exception\EntityNotFoundException
+     *
+     * @return \Generated\Shared\Transfer\ApiItemTransfer
+     */
+    public function get(int $id): ApiItemTransfer
+    {
+        $orderTransfer = $this->salesFacade->findOrderByIdSalesOrder($id);
 
-            throw new EntityNotSavedException('Could not save order: ' . implode(',', $errors));
+        if ($orderTransfer === null) {
+            throw new EntityNotFoundException(
+                'Could not found order.',
```

Comment:
Consider lingo change to:

1. Order not found. 
or
2. Couldn't find order. / Could not find order.